src/accessibility.cs

```diff
@@ -53,4 +140,173 @@ interface AXCustomContentProvider
+		AXDataPointValue[] AdditionalValues { get; set; }
+
+		[NullAllowed, Export ("label")]
+		string Label { get; set; }
+
+		[NullAllowed, Export ("attributedLabel", ArgumentSemantic.Copy)]
+		NSAttributedString AttributedLabel { get; set; }
+
+		[Export ("initWithX:y:")]
+		IntPtr Constructor (AXDataPointValue xValue, [NullAllowed] AXDataPointValue yValue);
+
+		[Export ("initWithX:y:additionalValues:")]
+		IntPtr Constructor (AXDataPointValue xValue, [NullAllowed] AXDataPointValue yValue, [NullAllowed] AXDataPointValue[] additionalValues);
+
+		[Export ("initWithX:y:additionalValues:label:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (AXDataPointValue xValue, [NullAllowed] AXDataPointValue yValue, [NullAllowed] AXDataPointValue[] additionalValues, [NullAllowed] string label);
+	}
+
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AXDataPointValue : NSCopying
+	{
+		[Export ("number", ArgumentSemantic.Assign)]
+		double Number { get; set; }
+
+		[Export ("category")]
+		string Category { get; set; }
+
+		[Static]
+		[Export ("valueWithNumber:")]
+		AXDataPointValue ValueWithNumber (double number);
+
+		[Static]
+		[Export ("valueWithCategory:")]
+		AXDataPointValue ValueWithCategory (string category);
+	}
+
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AXDataSeriesDescriptor : NSCopying
+	{
+		[NullAllowed, Export ("name")]
+		string Name { get; set; }
+
+		[Export ("attributedName", ArgumentSemantic.Copy)]
+		NSAttributedString AttributedName { get; set; }
+
+		[Export ("isContinuous")]
+		bool IsContinuous { get; set; }
+
+		[Export ("dataPoints", ArgumentSemantic.Copy)]
+		AXDataPoint[] DataPoints { get; set; }
+
+		[Export ("initWithName:isContinuous:dataPoints:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string name, bool isContinuous, AXDataPoint[] dataPoints);
+
+		[Export ("initWithAttributedName:isContinuous:dataPoints:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (NSAttributedString attributedName, bool isContinuous, AXDataPoint[] dataPoints);
+	}
+
+	[Watch (8,0), NoTV, NoMac, iOS (15,0)]
+	[Flags]
+	[Native]
+	public enum AXHearingDeviceEar : ulong
+	{
+		None = 0,
+		Left = 1 << 1,
+		Right = 1 << 2,
+		Both = Left | Right,
+	}
+
+	[Watch (8,0), NoTV, NoMac, iOS (15,0)]
+	[Static]
+	[Partial]
+	partial interface AXHearingUtilities
+	{
+		[Field ("AXMFiHearingDeviceStreamingEarDidChangeNotification")]
+		NSString StreamingEarDidChangeNotification { get; }
+
+		[Field ("AXMFiHearingDevicePairedUUIDsDidChangeNotification")]
+		NSString PairedUUIDsDidChangeNotification { get; }
+	}
+
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	[BaseType (typeof(NSObject))]
+	interface AXLiveAudioGraph
+	{
+		[Static]
+		[Export ("start")]
+		void Start ();
+
+		[Static]
+		[Export ("updateValue:")]
+		void UpdateValue (double value);
```

Comment:
-> `Update` since `Value:` is the parameter (double check with Swift name when unsure)

_update: swift uses `updateValue` but I still think `Update` is better :)_

Reply:
haha, sounds good!!! :D